src/System.IO.FileSystem/tests/PortedCommon/ReparsePointUtilities.cs

```diff
@@ -39,7 +39,9 @@ public static bool CreateSymbolicLink(string linkPath, string targetPath)
         {
             symLinkProcess = Process.Start("cmd", string.Format("/c mklink \"{0}\" \"{1}\"", linkPath, targetPath));
         }
-        else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+        else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ||
+                 RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD")) ||
+                 RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
Could you just change this to be an else rather than an else if?  If there's ever a non-Windows platform we deal with that doesn't have ln with this syntax, we can deal with it then.